### DISCUSSION TITLE
Progress Steps - Design and Spec RFC

### DISCUSSION THREAD

**Discussion by loreina** [twilio-labs/paste#3223] [Review request]
_Attention engineers: if you haven't yet had a chance to review the proposed Progress Steps API, ➡️  [take a look at that RFC here](https://github.com/twilio-labs/paste/discussions/3207) ⬅️ and feel free to leave your feedback!_

# Progress Steps - Design and Spec RFC

Hi all, Paste is adding a Progress Steps component and we would like your input. 

<img width="633" alt="image" src="https://github.com/twilio-labs/paste/assets/23108291/c730fc8a-8e00-4964-94bc-0b2fddecc46e">

## Component spec

Review [the component spec](https://docs.google.com/document/d/1isFKqNTb7bynl2PIMtZjwGCNKrasdYVrrwm3mPm5xK4/edit) any leave any comments/suggestions/questions directly in the google doc.

## Design

Review the below design proposal and leave any comments in this thread or in the [Figma file](https://www.figma.com/file/l2qtpQVMmXzQPVB3rEAA8i/branch/NF3oEp5N2rxsTF3209H4fs/%5BNEW%5D-Paste-Components?type=design&node-id=28554-176700&t=8NV6fSWPGanGx5lC-0). 

![image](https://github.com/twilio-labs/paste/assets/23108291/b349178a-c827-4e08-abca-fdcda9513d12)

Thank you!

**Comment by TheSisb** [comment]
As the component is now released in core 19.2.0, I am marking this RFC closed.
